audio/out/ao_jack.c

```diff
@@ -212,11 +242,10 @@ static int init(struct ao *ao)
 
     ao->samplerate = jack_get_sample_rate(p->client);
 
-    jack_latency_range_t jack_latency_range;
-    jack_port_get_latency_range(p->ports[0], JackPlaybackLatency,
-                                &jack_latency_range);
-    p->jack_latency = (float)(jack_latency_range.max + jack_get_buffer_size(p->client))
-                      / (float)ao->samplerate;
+    p->buffer_size       = ATOMIC_VAR_INIT(0);
+    p->graph_latency_max = ATOMIC_VAR_INIT(0);
```

Comment:
Yes, my mistake. A look at the standard indicates that I am meant to use `atomic_init(&p, 0)` instead.

However, looking at the rest of the codebase, none of the uses of atomic types are initialised like this prior to use. I'm guessing the reason this hasn't caused problems before is because no reasonable implementation of `<stdatomic.h>` uses any additional state that needs to be initialised.

So we have two options:
1) I can make a patch to insert calls to `atomic_init` everywhere else in the code, for standards compliance.
2) I can remove these two lines.